test/jdk/java/nio/file/attribute/BasicFileAttributeView/CreationTime.java

```diff
@@ -141,6 +148,24 @@ static void test(Path top) throws IOException {
         }
     }
 
+    /**
+     * read the output of linux command `stat -c "%w" file`, if the output is "-",
+     * then the file system doesn't support birth time
+     */
+    public static boolean supportBirthTimeOnLinux(Path file) {
+        try {
+            String filePath = file.toAbsolutePath().toString();
+            ProcessBuilder pb = new ProcessBuilder("stat", "-c", "%w", filePath);
+            pb.redirectErrorStream(true);
+            Process p = pb.start();
+            BufferedReader b = new BufferedReader(new InputStreamReader(p.getInputStream()));
+            String l = b.readLine();
+            if (l != null && l.equals("-")) { return false; }
+        } catch(Exception e) {
```

Comment:
There are several bugs/issues with this method, did you mean to bring it back? If we are launching subprocesses in this test then it can use ProcessTools infrastructure.

Reply:
It's difficult to enumerate all the combination of file system type and linux kernel version to determine the testd file support birth time or not, So I want to bring it back.
Thanks for the advice, I will use `ProcessTools infrastructure` later.

Reply:
An alternative could be to use FFM API to query the statx bufs mask the kernel returns testing for the birth time bit like the actual code does.

Reply:
I think this PR needed to backport to jdk21u and jdk17u in future. Maybe in jdk17u can't use`FFM API`?

Reply:
Maybe we can use this solution https://github.com/openjdk/jdk21u/pull/282

Reply:
The solution has been change to, get the result from `statx` linux syscall througt `FFM API`, to determine the testd file support birth ot not.